circle.yml

```diff
@@ -9,12 +9,11 @@ dependencies:
   override:
     # Used by container_spec
     - docker pull alpine
-    - docker build -t=$PRIVATE_REGISTRY/$CIRCLE_PROJECT_REPONAME:b$CIRCLE_BUILD_NUM .
-    - bundle install
+    - make image
 
 test:
   override:
-    - bundle exec rake
+    - make test
```

Comment:
@dblandin I agree with that, and that's why I put it in there original. Unfortunately, I don't want to take it out of the Makefile because then I'm sure there will be a day, or many days, I run `make test` and can't figure out why my changes aren't having the desired effect.

I'd probably go back to what I had. Even though it's technically duplicated work, it is a 100% cached and fast build when Circle runs the test. I don't feel strongly though. @wfleming ?